### PR TITLE
[jasmine] fix bad types for set default spy strategy

### PR DESCRIPTION
**Pull request by chivesrs**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://jasmine.github.io/tutorials/default_spy_strategy
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/jasmine/index.d.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@
 //                 kirjs <https://github.com/kirjs>
 //                 Md. Enzam Hossain <https://github.com/ienzam>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.1
```
Comment (devoto13) [inline types/jasmine/index.d.ts:21]: According to [README](https://github.com/DefinitelyTyped/DefinitelyTyped#older-versions-of-typescript-33-and-earlier) DT only supports TypeScript >=3.4. Can you explain why is this change necessary?

Comment (chivesrs) [reply types/jasmine/index.d.ts:21]: npm test jasmine fails without a comment like this, and this was the suggested comment when I ran said command

If DT wants to support minimum 3.4 then they need to fix their test tooling first

Comment (devoto13) [reply types/jasmine/index.d.ts:21]: Interesting. Maybe you have outdated `dtslint` package or something? I don't get this error with:

```
$ rm -rf node_modules
$ npm i
$ npm test jasmine
```

Comment (chivesrs) [reply types/jasmine/index.d.ts:21]: Yeah I figured that... I'll update and try

Comment (chivesrs) [reply types/jasmine/index.d.ts]: That was it. Force pushed and fixed.

**Review by devoto13 — APPROVED** [review]
